### PR TITLE
Feat/bulk methylation templates

### LINKED ISSUE #67: Import bisulfite-sequencing template (level 2/3)
These are known as BulkMethylation-seq.

### PR DESCRIPTION
**Pull request by anngvu**
- Close #67: Added templates based on HTAN version, but removed a couple of optional attributes to keep more streamlined.
- Also throw in #79: Easier testing, prepare us for new December setup.

### CODE REVIEW

File: modules/classes/sequencing.yaml [hunk 1]
```diff
@@ -1,15 +1,23 @@
 classes:
   
-  Sequencing:
+  File:
```
Comment (cconrad8) [inline modules/classes/sequencing.yaml:3]: Wondering if we want to change imaging to inherit from "File" as well.

**Review by cconrad8 — COMMENTED** [review]
Looks good to me! Or we can make additional changes to imaging now re: my File comment

File: modules/classes/sequencing.yaml (hunk 1 above)
Comment (cconrad8) [reply modules/classes/sequencing.yaml:3]: Might not be as easy as I think though..

Comment (cconrad8) [reply modules/classes/sequencing.yaml:3]: OK after trial and errors..I just changed LabID to ParticipantID to be consistent with the other templates..I think that's all for now - 

General question on the inheriting - Can we do "is_a" across different yaml files? (Like have imaging inherit from File if it's in another yaml file?)

Comment (anngvu) [reply modules/classes/sequencing.yaml:3]: OK, thanks for updating LabID to make it more consistent. Yep, can do `is_a` across different files.

Comment (anngvu) [reply modules/classes/sequencing.yaml:3]: We'll apply the refactoring for imaging in another PR to keep this one smaller.

**Pull request merged**